CONTRIBUTING.md

```diff
@@ -55,6 +55,17 @@ A note to submitters: If you need multiple people to see your PR before it's mer
 
 If you're not sure, ask for a second opinion in #tts-handbook first.
 
+### Using repo specific labels
```

Comment:
Might be worth adding a graf that these aren't required, but can be helpful signals to other contributors.